core/indexer/common.go

```diff
@@ -304,26 +305,33 @@ func prepareBufferMiniblocks(buff bytes.Buffer, meta, serializedData []byte) byt
 	return buff
 }
 
-func serializeBulkTxs(
-	bulk []*Transaction,
+func serializeTransactions(
+	transactions []*Transaction,
 	selfShardID uint32,
 	getAlreadyIndexedItems func(hashes []string, index string) (map[string]bool, error),
-) bytes.Buffer {
-	var buff bytes.Buffer
+) []bytes.Buffer {
 	var err error
 
-	txsHashes := make([]string, len(bulk))
-	for idx := range bulk {
-		txsHashes[idx] = bulk[idx].Hash
+	txsHashes := make([]string, len(transactions))
+	for idx := range transactions {
+		txsHashes[idx] = transactions[idx].Hash
 	}
 
-	existsInDb, err := getAlreadyIndexedItems(txsHashes, txIndex)
-	if err != nil {
-		log.Warn("indexer", "error", err.Error())
-		return buff
+	existsInDb := make(map[string]bool)
+	bulksTxsHashes := buildBulksOfHashes(txsHashes)
+	for i := 0; i < len(bulksTxsHashes); i++ {
+		exitsInDbBulk, err := getAlreadyIndexedItems(bulksTxsHashes[i], txIndex)
```

Comment:
`err` shadows above declaration, can you please rename it here?

Reply:
renamed